src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -1,100 +1,38 @@
 import { CompositeLayer } from '@deck.gl/core';
 // eslint-disable-next-line import/extensions
 import VivViewerLayerBase from './VivViewerLayerBase';
-import {
-  padWithDefault,
-  isInTileBounds,
-  DEFAULT_COLOR_OFF,
-  MAX_COLOR_INTENSITY
-} from './utils';
-
-const MAX_SLIDERS_AND_CHANNELS = 6;
+import { isInTileBounds } from './utils';
+import { padColorsAndSliders } from '../utils';
```

Comment:
```suggestion
import { isInTileBounds, padColorsAndSliders } from './utils';
```

Reply:
They are in different files now, I think.